rules_conditions.md

```diff
@@ -0,0 +1,226 @@
+# Refinery Conditions
+
+## Overview
+
+Refinery rules are described as a series of conditions.
+Each condition is composed from a combination of these named parameters:
+
+- `Field` (or `Fields`)
+- `Operator`
+- `Value`
+- `Datatype`
+
+The `Operator` is never optional, and controls which of the parameters are required and which are optional.
```

Comment:
1. If the `Operator` is never optional, doesn't that mean it's required?

```suggestion
The `Operator` is a required parameter, and controls which of the condition's other parameters are required and which are optional.
```

Reply:
For the second half of the sentence, maybe replace "other parameters" with "other elements"?